hermes/shared/file_gen.go

```diff
@@ -7,11 +7,9 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-// Author: Alicja Kwiecinska (kwiecinskaa@google.com) github: alicjakwie
-
+//
 // File generation for Hermes
-package hermes
+package shared
```

Comment:
Change the file path as well.

Reply:
The path for this file has been changed. It is now in the directory `shared`, so its path is: `step224-2020/hermes/shared/file_gen.go`.

Reply:
"shared" has a code-smell (it's similar to "misc" and "util"). What's wrong with "hermes"?

Reply:
We can move it to the "hermes" package. I'll adjust the file path accordingly.